test/acts_as_authentic_test/password_test.rb

```diff
@@ -92,6 +92,23 @@ def test_transitioning_password
       )
     end
 
+    def test_password_changed_false_if_transitioning
+      User.class_eval do
+        after_update :verify_not_changed, unless: :password_changed?
+        def verify_not_changed; end
+      end
+      ben = users(:ben)
+      mock_method = MiniTest::Mock.new
+      mock_method.expect :call, nil
+      ben.stub :verify_not_changed, mock_method do
+        transition_password_to(Authlogic::CryptoProviders::BCrypt, ben)
+      end
+      mock_method.verify
+      User.class_eval do
+        User.skip_callback :update, :after, :verify_not_changed
+      end
```

Comment:
Would something simpler work too, vs using callbacks? I don't think the record would get reloaded, so we could do e.g. 

``` ruby
     ben = transition_password_to(Authlogic::CryptoProviders::BCrypt, users(:ben)).first
     refute ben.send(:password_changed?)`
```

Or, maybe we should just move this assertion into `transition_password_to`?